src/containers/NdlaFilm/components/NdlaFilmArticle.tsx

```diff
@@ -39,18 +39,19 @@ const NdlaFilmArticle = ({ fieldName, onUpdateArticle }: Props) => {
   const { t } = useTranslation();
   const form = useFormikContext();
   const [field] = useField<string>(fieldName);
+  const [selectedArticle, setSelectedArticle] = useState<undefined | IArticleV2>(undefined);
 
-  const article = useArticleSearch(
-    { ids: field.value ? `${getIdFromUrn(field.value)}` : undefined },
-    { enabled: !!field.value },
-  );
-
-  const selectedArticle: undefined | IArticleSummaryV2 = useMemo(() => {
-    if (article.isLoading || !article.data) {
-      return undefined;
-    }
-    return article.data.results[0];
-  }, [article.data, article.isLoading]);
+  useEffect(() => {
+    const initSelectedArticle = async () => {
+      if (field.value) {
+        const response = await getArticle(getIdFromUrn(field.value));
```

Comment:
Som sagt har du ingen urn å hente id fra. Dette burde være en rein id.

Reply:
Jammen kjem det en urn, men den blir i tilfelle lagt på en anna plass enn i søket. jaja.

Reply:
Stemmer; sida datatypen i backend var string, og sida attributtet var navngitt article, har jeg tenkt at det var en urn som skulle lagres for artikkelen (og antatt at hvis det var en id vi ville lagre, hadde attributtet hatt navn articleid og typen vært number). Derfor er det også urn jeg fyrer inn når valg gjøres i søkelista, på AsyncDropdown-komponenten, her
```javascript
onChange={(article: IArticleSummaryV2) =>
    onUpdateArticle(field, form, getUrnFromId(article.id))
}
```
Så at navngivning hang igjen på update-funksjonen som articleid, så den har jeg også nå retta nå. Alternativt, før jeg fortsetter og dette merges inn, var det meninga at det skulle være id som skulle lagres, og ikke urn? Og burde backend da isåfall også vært oppdatert for å reflektere det (navngivning + type)?

Reply:
Kan godt la det være slik. Urn gir meining for alt anna på ndla-film fordi der peiker det på ressurser i taksonomi i faget ndla film. Akkurat denne artikkelen ligger ikkje i faget men er heilt utenfor så den kunne hatt en id lagra. Men la det være som du har laga det.